src/components/Footer.tsx

```diff
@@ -1,50 +1,90 @@
 import { ReactElement } from "react";
+import ChevronRightIconSVG from ".././assets/ChevronRightIcon.svg";
+import GithubIconSVG from ".././assets/GithubIcon.svg";
+import InstagramIconSVG from ".././assets/InstagramIcon.svg";
+import InternetWebIconSVG from ".././assets/InternetWebIcon.svg";
 import ".././styles.css";
 
 // Footer with source code and project information that goes at the bottom of each page
-const Footer = (): ReactElement => {
+const Footer = (props: { hideInfo: boolean }): ReactElement => {
   return (
-    <div className="w-full px-2 py-4 mt-4 flex flex-row justify-between border-t-2 bottom-0">
-      <div>
-        Made by students @&nbsp;
-        <a
-          href="https://sandboxnu.com"
-          className="link"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Sandbox
-        </a>
-        . Source on&nbsp;
-        <a
-          href="https://github.com/sandboxnu/sga-tooling"
-          className="link"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          GitHub.
-        </a>
-      </div>
+    <div className="w-full px-6 py-6 mt-4 flex flex-col justify-between bottom-0 bg-sga-red md:py-6 md:px-8 md:items-center">
+      <div className="flex flex-row justify-between items-center pb-1">
+        <div className="font-montserrat text-xl font-bold text-white">
+          Made by Sandbox
+        </div>
 
-      <div>
-        <a
-          href="https://github.com/sandboxnu/office-hours/discussions"
-          className="link"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Give us feedback
-        </a>
-        <span> or </span>
-        <a
-          href="https://github.com/sandboxnu/office-hours/issues"
-          className="link"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          File a bug report
-        </a>
+        {props.hideInfo ? (
```

Comment:
Instead of having two `props.hideInfo ?` statements, we can put it into one
```suggestion
        {props.hideInfo ? Show ChevronRightIcon : Show other thing}
```